src/storage/page_buffer.c

```diff
@@ -7709,6 +7709,8 @@ pgbuf_claim_bcb_for_fix (THREAD_ENTRY * thread_p, const VPID * vpid, PAGE_FETCH_
 	}
 #endif /* ENABLE_SYSTEMTAP */
 
+      pgbuf_request_data_page (vpid);
```

Comment:
Name suggestion to make function purpose more obvious: `pgbuf_request_data_page_from_page_server()`